charts/airbyte-pod-sweeper/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+name: pod-sweeper
+description: Helm chart to deploy airbyte-pod-sweeper
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: "0.39.36"
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.39.23-alpha"
```

Comment:
If we want to keep this appVersion in sync with the actual Airbyte Version, we can add some entries to [.bumpversion.cfg](https://github.com/airbytehq/airbyte/blob/master/.bumpversion.cfg) so that bump version will automatically increment all these versions whenever the Airbyte Version is updated.

As for when the Airbyte Version is updated, that happens whenever devs run the [Release Open Source Airbyte](https://github.com/airbytehq/airbyte/actions/workflows/release-airbyte-os.yml) workflow.
One of the actions that workflow does is [call bumpversion](https://github.com/airbytehq/airbyte/blob/master/tools/bin/release_version.sh#L29), which will use the `.bumpversion.cfg` to find version strings to update and then create a commit with all the version bumps.
Here's an example of a PR that is created whenever a dev runs the release workflow: https://github.com/airbytehq/airbyte/pull/14719/files

Reply:
Updated bumpconfig